src/main/java/org/skriptlang/skript/bukkit/whitelist/elements/EvtWhitelist.java

```diff
@@ -0,0 +1,91 @@
+package org.skriptlang.skript.bukkit.whitelist.elements;
+
+import com.destroystokyo.paper.event.server.WhitelistToggleEvent;
+import io.papermc.paper.event.server.WhitelistStateUpdateEvent;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptEvent;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.skript.registrations.EventValues;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+
+public class EvtWhitelist extends SkriptEvent {
+
+	static {
+		Skript.registerEvent("Whitelist", EvtWhitelist.class, CollectionUtils.array(WhitelistToggleEvent.class, WhitelistStateUpdateEvent.class),
+				"server whitelist [state:(:enable[d]|disable[d])]",
```

Comment:
`on server whitelist` doesn't make much sense.
Likewise, `on player whitelist added` is really clunky. I'd prefer `on whitelist addition/removal of player` or something similar.